app/views/content/non-uk-teachers/international-qualified-teacher-status.md

```diff
@@ -127,24 +122,20 @@ The following providers specialise in particular regions, or should be contacted
 
 * [University of Portsmouth](https://www.port.ac.uk/study/short-courses-and-cpd/international-qualified-teacher-status-iqts-distance-learning) – contact this provider for more information
```

Comment:
Don't think we need the 'contact this provider for more info? As surely that statement applied to all of them?